src/scripts/main.js

```diff
@@ -1 +1,78 @@
 'use strict';
+
+let isClick = false;
+let isContextmenu = false;
+
+function getNotification(className, message) {
+  const notification = document.createElement('div');
+
+  notification.setAttribute('data-qa', 'notification');
+  notification.className = className;
+  notification.textContent = message;
+  document.body.append(notification);
+}
+
+const firstPromise = new Promise((resolve, reject) => {
+  const click = addEventListener('click', () => {
```

Comment:
The `click` event listener is removed immediately after being added, which means it will not function as intended. Consider removing the listener only after the promise is resolved or rejected.